db/migrate/20230210225746_add_form_template_to_categories.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddFormTemplateToCategories < ActiveRecord::Migration[7.0]
+  def change
+    add_column :categories, :form_template, :json, null: true
+  end
```

Comment:
Want to add `:limit` to column, but I'm getting errors. Is this possible with the `:json` type?